src/middlewared/middlewared/plugins/smb_/sharesec.py

```diff
@@ -44,15 +44,13 @@ class Config:
         'data_type': 'BYTES'
     }
 
-    @private
```

Comment:
If you are removing private from this (and others), then should some documentation be provided?

TBH, I'd lean more towards keeping these private.  Is there a reason you're changing it?

Reply:
At some point in the past I made the whole plugin private. These are redundant.